src/quacc/runners/prep.py

```diff
@@ -77,6 +77,9 @@ def calc_setup(
         if isinstance(copy_files, (str, Path)):
             copy_files = {copy_files: "*"}
 
+        if isinstance(copy_files, list):
+            copy_files = {source: "*" for source in copy_files}
```

Comment:
This new behavior requires a dedicated unit test.

Reply:
Worth noting: even without these lines in `prep.py` espresso recipes would work fine because we transform the `list[SourceDirectory]` into a `dict[SourceDirectory, Filenames]` in `quacc.recipes.espresso._base._prepare_calc`

I added it here "just in case".